xbmc/interfaces/json-rpc/AudioLibrary.cpp

```diff
@@ -1022,7 +1022,7 @@ JSONRPC_STATUS CAudioLibrary::SetSongDetails(const std::string &method, ITranspo
   if (!musicdatabase.UpdateSong(song, updateartists))
     return InternalError;
 
-  CJSONRPCUtils::NotifyItemUpdated();
+  CJSONRPCUtils::NotifyItemUpdated(song);
```

Comment:
I would introduce a method taking a const CFileItem& - so you get a generic function that can be used to notify whatever a file item can carry.